auth/src/main/java/com/firebase/ui/auth/provider/FacebookProvider.java

```diff
@@ -43,7 +43,10 @@
 import java.util.List;
 
 public class FacebookProvider implements IdpProvider, FacebookCallback<LoginResult> {
+    public static final String PUBLISH_PERMISSION = "publish";
```

Comment:
Ok so Facebook's permissions docs leave something to be desired but I have a few questions:

  * Looks like the `publish_actions` permissions already acts as the signaling value here.  You need that permission to do any publishing (https://developers.facebook.com/docs/facebook-login/permissions#reference-publish_actions)
  * Facebook recommends against asking for publish permissions at login time, so not even sure we should offer this (but I guess it can't hurt)
  * What happens if we get the split wrong?  Can you try requesting a bunch of read-only permissions using the `loginWithPublish` method and see what happens?